config/idm-endpoints/scripts-content/companyauth.js

```diff
@@ -444,21 +469,18 @@
             }
         };
     }
-    else if (request.action === RequestAction.SET_STATUS_BY_USERNAME) {
+    else if (request.action === RequestAction.INVITE_USER_BY_USERNAME) {
 
-        log("Set membership status request by username");
+        log("Request to set membership status to PENDING (send invitation) by subject userName");
 
-        var actor = getUserById(request.content.callerId);
-        var isAdminUser = JSON.stringify(actor.authzRoles).indexOf(INTERNAL_IDM_ADMIN) !== -1;
-
-        if (!request.content.callerId || !request.content.subjectUserName || !request.content.companyNumber || !request.content.status) {
-            log("Invalid parameters - Expected: callerId, subjectUserName, companyNumber, status");
-            throw { code: 400, message: "Invalid Parameters - Expected: callerId, subjectUserName, companyNumber, status" };
+        if (!request.content.subjectUserName || !request.content.companyNumber || !request.content.status) {
+            log("Invalid parameters - Expected: subjectUserName, companyNumber, status");
+            throw { code: 400, message: "Invalid Parameters - Expected: subjectUserName, companyNumber, status" };
         }
 
         if (request.content.status !== AuthorisationStatus.PENDING) {
-            log("Invalid parameters - The status can only be 'pending'");
-            throw { code: 400, message: "Invalid Parameters - The status can only be 'pending'" };
+            log("Invalid parameters - The status for an invitation request can only be 'pending' for invitation requests");
```

Comment:
Duplication in sentence - maybe change to `Invalid parameters - The status for an invitation request can only be 'pending'`